app/models/subscription.rb

```diff
@@ -7,6 +7,10 @@ class Subscription < ApplicationRecord
   enum subscription_type: %w(Monthly Bi-Monthly Weekly)
 
   def set_subscription_type
-    self.subscription_type ||= 0
+    self.subscription_type ||= 2
+  end
+
+  def next_shipment
+    shipments.last
```

Comment:
This was the simplest way to grab the upcoming shipment for the FE without having to deal with time